assets/scss/layout.scss

```diff
@@ -119,7 +119,7 @@ th {
     word-wrap: break-word;
   }
 
-  .box .nav-stacked > li.item > span.matches {
+  .nav-stacked > li.item > span.matches {
```

Comment:
This is based on https://github.com/sonata-project/SonataAdminBundle/commit/01677b25381aa3c3d87dfaf7fe8699d9bc4744f3#diff-711b84885f44c2228692b36af9383305c888a95422d4215fedc009ef4d6c944b

Reply:
Can you compile assets?

```
// make sure you have node 14
yarn install --frozen-lockfile
yarn encore production
```

Reply:
Also, you should put and & before the dot. Because the class is: .sonata-search-result-list.nav-stacked

Reply:
done, thanks! I've used node 15